libs/shared-atomic-design-components/src/lib/atoms/Checkbox.tsx

```diff
@@ -5,16 +5,18 @@ import './Checkbox.scss'
 
 interface Props {
   name: string
-  value: string
+  value?: string
   checked: boolean
-  children: React.ReactNode
-  disabled: boolean
+  children?: React.ReactNode
+  disabled?: boolean
+  handleChange?: (...args: any[]) => void
+  handleBlur?: (...args: any[]) => void
+  isSubmitting?: boolean
```

Comment:
What is `isSubmitting` for? In the storybook, it acts exactly as `disabled`. If it's redundant, how about using `disabled` only for all cases? Let me know if I'm missing something 🙏